pulp_container/app/authorization.py

```diff
@@ -205,19 +207,10 @@ def has_view_catalog_permissions(self, path):
         """
         Check if the authenticated user is an administrator and is requesting the catalog endpoint.
         """
-        try:
-            distribution = ContainerDistribution.objects.get(base_path=path)
-        except ContainerDistribution.DoesNotExist:
-            distribution = None
-
-        # Fake the request
-        request = Request(HttpRequest())
-        request.method = "GET"
-        request.user = self.user
-        request._full_data = {"base_path": path}
-        # Fake the corresponding view
-        view = namedtuple("FakeView", ["action", "get_object"])("catalog", lambda: distribution)
-        return self.access_policy.has_permission(request, view)
+        if path == "catalog" and self.user.is_staff:
+            return True
+        else:
+            return False
```

Comment:
This change is reverting to an old behavior, where we had the permission hard coded.
I think we want to keep this going through the access_policy.